src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -68,6 +68,23 @@
           maxlength="50"
           class="my-8"
         ></v-text-field>
+        <v-row>
+          <v-checkbox
+            v-model="officialTranscriptRequested"
+            :class="{ 'error-message': !atLeastOneChecked }"
+            color="primary"
+            label="I have requested the official transcript from my education institution"
+          ></v-checkbox>
+          <v-checkbox
+            v-model="officialTranscriptReceived"
+            :class="{ 'error-message': !atLeastOneChecked }"
+            color="primary"
+            label="The ECE Registry already has my official transcript for the course/program relevant to this application and certificate type"
+          ></v-checkbox>
+          <div v-if="!atLeastOneChecked" class="v-messages error-message" role="alert">Please select at least one Option</div>
```

Comment:
Should this be a rule?

Reply:
using :rule to validate. added div to toggle error message for now